crates/dotrain/src/parser/mod.rs

```diff
@@ -265,10 +265,34 @@ pub(crate) fn deep_read_quote<'a>(
     }
 }
 
+/// Parse a single key-value pair from cli arg
+pub fn parse_cli_key_val(
```

Comment:
doesn't this exist in clap?

Reply:
no, they have a similar fn in their docs for parsing `=` separated string args

Reply:
@rouzwelt can you give me an example of the difference?

Reply:
I mean in their docs, they have explained to use such fns as this one for parsing the =, not that they have that fn implemented in their code,
like in here
https://docs.rs/clap/latest/clap/builder/struct.ValueParser.html#example-1

Reply:
@rouzwelt i reckon we link direct to the clap docs in this code